resources/public/api/conf/1.0/application.yaml

```diff
@@ -761,8 +973,7 @@ components:
       description: The media type of the request body.
       schema:
         type: string
-        enum:
-          - application/xml
+        enum: ["application/vnd.hmrc.1.0+xml", "application/xml"]
```

Comment:
If we are not accepting application/xml on the new endpoint we need to ensure the api spec is not showing it as a valid value. Currently it is.

Reply:
I should not use this one. Added new Accept header reference